physics/docs/pdftxt/CU_GF_deep.txt

```diff
@@ -17,23 +17,23 @@ a spectrum of plumes within the grid box. Forcing is different for each characte
 from the PDFs. Aerosol dependence (considered experimental and not supported in this release) is implemented through dependence of rain generation and 
 evaporation formulations depending on the cloud concentration nuclei at cloud base (Berry 1968 \cite berry_1968, 
 Jiang et al.(2010) \cite Jiang_2010, and Lee and Feingold (2010) \cite lee_and_feingold_2010 ). Wet scavenging is considered to add a memory impact. GF is able to transport tracers.
-Recently, GPU capabilities and cap suppressing (\p do_cap_suppress) based on radar data assimilation have been added,and they are used only for the RAP suite.
+Recently, GPU capabilities and cap suppressing (\p do_cap_suppress) based on radar data assimilation have been added, and they are used only for the RAP suite.
 
-The impacts of GF scheme in operational RAP/HRRR include:(a)uses mass-flux schemes, which are more physically realistic than (sounding) adjustment schemes;
+The impacts of GF scheme in operational RAP/HRRR include: (a)uses mass-flux schemes, which are more physically realistic than (sounding) adjustment schemes;
 (b)takes parameterization uncertainty into account by allowing parameters from multiple convective schemes which can be perturbed 
-internally or with temporal and spatial correlation patterns; (c)for higher resolutions (less than 10 km), in addition to scale awareness as in Arakawa et al. (2011) \cite Arakawa_2011 GF can transition as grid spacing decreases into a shallow convection scheme; (d)Coupled to the grid scale precipitation and radiation schemes through passing of diagnosed cloud liquid and ice from simulated precipitating convective cloud and shallow convective clouds.
+internally or with temporal and spatial correlation patterns; (c)for higher resolutions (less than 10 km), in addition to scale awareness as in Arakawa et al. (2011) \cite Arakawa_2011 GF can transition as grid spacing decreases into a shallow convection scheme; (d)Coupled to the grid-scale precipitation and radiation schemes through passing of diagnosed cloud liquid and ice from simulated precipitating convective cloud and shallow convective clouds.
 
 \section version_cugf_enh CCPP Physics Updates
 
 \version UFS-SRW v3.0.0
 
-\b The \b Implementation \b of \b GF \b in \b RRFS
+<b>The Implementation of GF in RRFS prototypes</b>
```

Comment:
I added "prototypes" here due to the continued uncertainty around RRFS implementations